python/test/t_IntegrationExpansion_std.py

```diff
@@ -0,0 +1,149 @@
+#! /usr/bin/env python
+
+import openturns as ot
+import openturns.experimental as otexp
+import math as m
+
+ot.TESTPREAMBLE()
+
+
+# Problem parameters
+dimension = 3
+a = 7.0
+b = 0.1
+# Reference analytical values
+meanTh = a / 2
+covTh = (b**2 * m.pi**8) / 18.0 + (b * m.pi**4) / 5.0 + (a**2) / 8.0 + 1.0 / 2.0
+sob_1 = [
+    (b * m.pi**4 / 5.0 + b**2 * m.pi**8 / 50.0 + 1.0 / 2.0) / covTh,
+    (a**2 / 8.0) / covTh,
+    0.0,
+]
+sob_2 = [0.0, (b**2 * m.pi**8 / 18.0 - b**2 * m.pi**8 / 50.0) / covTh, 0.0]
+sob_3 = [0.0]
+sob_T1 = [
+    sob_1[0] + sob_2[0] + sob_2[1] + sob_3[0],
+    sob_1[1] + sob_2[0] + sob_2[2] + sob_3[0],
+    sob_1[2] + sob_2[1] + sob_2[2] + sob_3[0],
+]
+sob_T2 = [
+    sob_2[0] + sob_2[1] + sob_3[0],
+    sob_2[0] + sob_2[2] + sob_3[0],
+    sob_2[1] + sob_2[2] + sob_3[0],
+]
+sob_T3 = [sob_3[0]]
+# Create the Ishigami function
+inputVariables = ["xi1", "xi2", "xi3"]
+formula = ot.Description(1)
+formula[0] = (
+    "sin(xi1) + (" + str(a) + ") * (sin(xi2)) ^ 2 + (" + str(b) + ") * xi3^4 * sin(xi1)"
+)
+model = ot.SymbolicFunction(inputVariables, formula)
+
+# Create the input distribution
+distribution = ot.ComposedDistribution([ot.Uniform(-m.pi, m.pi)] * dimension)
+
+# Create the orthogonal basis
+enumerateFunction = ot.LinearEnumerateFunction(dimension)
+productBasis = ot.OrthogonalProductPolynomialFactory(
+    [ot.LegendreFactory()] * dimension, enumerateFunction
+)
+
+# Create the projection strategy
+samplingSize = 250
+listProjectionStrategy = list()
+# Monte Carlo sampling
+inputSample = ot.LowDiscrepancyExperiment(
+    ot.SobolSequence(), distribution, samplingSize
+).generate()
+outputSample = model(inputSample)
+degree = 6
+basisSize = enumerateFunction.getStrataCumulatedCardinal(degree // 2)
+algo = otexp.LeastSquaresExpansion(
+    inputSample,
+    outputSample,
+    distribution,
+    productBasis,
+    basisSize,
+)
+ot.RandomGenerator.SetSeed(0)
+algo.run()
+
+# Examine the results
+result = algo.getResult()
+print("###################################")
+# print "coefficients=", result.getCoefficients()
+residuals = result.getResiduals()
+print("residuals=", residuals)
+relativeErrors = result.getRelativeErrors()
+print("relativeErrors=", relativeErrors)
+
+# Post-process the results
+vector = ot.FunctionalChaosRandomVector(result)
+mean = vector.getMean()[0]
+print("mean=%.8f" % mean, "absolute error=%.10f" % abs(mean - meanTh))
```

Comment:
Please use `assert_almost_equal` functions. You have examples of this in the code for the PR of the Sobol' indices [t_FunctionalChaosSobolIndices_std.py](https://github.com/mbaudin47/openturns/blob/32cdf26a4eb6076ad9db59a72530f1ea4bc2d137/python/test/t_FunctionalChaosSobolIndices_std.py#L67)